src/q4/complanal-INMA1315/exam/2018/Juin/Majeure/complanal-INMA1315-exam-2018-Juin-Majeure.tex

```diff
@@ -1,9 +1,12 @@
 \documentclass[fr]{../../../../../../eplexam}
 
-\hypertitle{Compléments d'Analyse}{6}{INMA}{1315}{2018}{Juin}{Majeure}
+\hypertitle{Compléments d'analyse}{4}{INMA}{1315}{2018}{Juin}{Majeure}
 {Nathan Jacques}
 {Michel Willem}
 
+\textbf{Attention}: le cours a changé de façon substantielle en 2019 et cet examen porte sur une ancienne version.
```

Comment:
I would've put this on this title page (i.e. `{Michel Willem}[\paragraph{Attention} Le cours ...]`, I think), so we have a clear separation with the content of the document. Same comment for the summary.

Reply:
Je viens de regarder ce que ça donne et j'ai l'impression que ça cache l'information plus qu'autre chose. C'est tout en bas de la page de garde et perso je ne la lis jamais. ^^

Reply:
Ça marche, le principal c'est que ce soit visible en effet.

> perso je ne la lis jamais
Tu connais donc pas encore les bons bails de procrastination en session...

Reply:
En général je suis trop à la bourre pour me permettre de procrastiner...

Reply:
Justement, c'est quand on est à la bourre qu'on procrastine le mieux ;)